pkg/object/objects.go

```diff
@@ -480,6 +483,27 @@ func manifestDiff(aom, bom map[string]*K8sObject) (string, error) {
 	return sb.String(), nil
 }
 
+func objectIgnorePaths(objectName string, ignoreResources string) []string {
```

Comment:
I'm thinking the function has something common with `filterResourceWithSelectAndIgnore` at
https://github.com/istio/operator/blob/312ab0dc3e25d50737fe7f64eb26e4f340127bc5/pkg/object/objects.go#L401-L437

Can we build a map with resource as key and ignorePaths as value from `filterResourceWithSelectAndIgnore` ?
then we don't need to iterate the `ignoreResources` again here.

Reply:
Extracted the maps as parameters for those 2 functions.